BUILD.md

```diff
@@ -0,0 +1,42 @@
+# Building barnabas
+
+Barnabas is currently using `make` as its main build system. Our make build supports several different targets mainly for building and pushing docker images.
```

Comment:
I would remove the 'currently', as it suggests that might change. While it might *indeed* change so might just about everything in this project, and we don't put currently in other bits of doc.